web/js/codeworld.js

```diff
@@ -585,9 +643,13 @@ function run(hash, dhash, msg, error) {
     if (hash) {
         window.location.hash = '#' + hash;
         document.getElementById('shareButton').style.display = '';
+        if (!!navigator.mediaDevices.getUserMedia) {
```

Comment:
At this point, the program may still have failed to build.  Perhaps you want to add this logic at the same place that `runner.contentWindow` is shown or hidden below.